src/ESM.sol

```diff
@@ -55,7 +56,7 @@ contract ESM {
         min = min_;
     }
 
-    function revokesGovernanceAccess() external view returns (bool ret) {
```

Comment:
was this a typo ?

Reply:
i don't think so looking at the git blame, i think it was just a different perspective on the tense.  I felt this made more sense since this function does not revoke access just reports on whether it has been revoked

Reply:
So the original name is actually correct.
The getter checks if the deployed version `revokes` or not access to governance not if was revoked.

The original intention of this `esm` is you can deploy it passing or not the `proxy` param. This getter is just an "easier" way to check how was deployed.

Reply:
I'm not sure I agree Gonza, when I hear/read: `revokes` governance I think that either calling the function will revoke governance, not that it is going to return a value about how it was deployed.  I guess since `fire` **will revoke** governance I can see an interpretation where this function tells the user whether or not the ESM will revoke governance in the future...

Either way, I don't think this is an important distinction.  I will revert the change and keep the original intent